spec/models/transaction_spec.rb

```diff
@@ -3,7 +3,6 @@
 RSpec.describe Transaction do
   describe 'validations' do
     it { should validate_presence_of(:credit_card_number) }
-    it { should validate_presence_of(:credit_card_expiration_date) }
```

Comment:
This seems to be because of the fact that the expiration date is blank in the CSV files. Maybe we should set a default value though?

Reply:
I don't think it is necessary, we do not need to have validations for every single column. I would prefer to leave it for now and see if we need to do anything with it in further user stories -but I am not against adding a default value if you would like.